src/virtualenv/seed/embed/pip_invoke.py

```diff
@@ -18,8 +18,8 @@ def __init__(self, options) -> None:
     def run(self, creator):
         if not self.enabled:
             return
-        for_py_version = creator.interpreter.version_release_str
-        with self.get_pip_install_cmd(creator.exe, for_py_version) as cmd:
+        self.warn_wheel_options(creator.interpreter)
```

Comment:
Hmm, I don't like how this ended up. Perhpas better to attach the interpreter version information onto the `options` [here](https://github.com/pypa/virtualenv/blob/main/src/virtualenv/run/__init__.py#L51):

```python
options.py_version = parser._interpreter.version_info  # noqa: SLF001
```

And keep all the logic inside the `__init__` as was before this PR 🤔

Reply:
Ok then, will do.

Reply:
See what I pushed. It does not work:

```
AttributeError: 'VirtualEnvOptions' object has no attribute 'py_version'
```

Reply:
Never mind, I got it.